### DISCUSSION TITLE
UI Suggestions

### DISCUSSION THREAD

**Discussion by Andrew-McCauley** [Rem0o/FanControl.Releases#650] [Ideas]
Brilliant software - I was happy enough to find something that would be able to respond to GPU fan speeds, let alone do a max operation to a CPU and GPU fan curve!

1. I'd love if there were a way to integrate speeds into controls. Unless I'm mistaken, fan speeds cards just contributes RPM numbers, which could be folded into controls. I guess it's not total overlap since you could have some speeds that can't be controlled or possibly even controls that can't give RPM readouts (or either that is hidden), but for many/most people I think it would be great to add the RPM numbers to the control cards in some way to save space and make things more clear.

2. For mix curves, it'd be great to see the contributing curves (possibly in different colours) and the resulting curve (in thicker black or white or whatever colour is neutral for the theme).

3. It'd be great to be able to drag the cards into whatever order you want, irrespective of what category they fall under.

**Comment by Rem0o** [comment]
1. That exists already. Kinda sad Jay didn't mention it in his video ;)  It's called "pairing".  Use the 3 dot menu on each control card, or do it from here: 
![image](https://user-images.githubusercontent.com/14118956/164983453-5d812b2b-218c-423b-bf6f-444715684b8b.png)

Then, hide all the speed section with the checkbox in the hamburger menu.

2. The mix card already shows which curve is active in max scenarios. How would that work if you got multiple mixes using the same curves?

3. Well, that I can't really, more of a technical/backend reason, but it would be a total pain if collections could be messed with that way.

**Reply by Andrew-McCauley** [reply]
1. Already did that, looks like it didn't work then and I retried it and it failed again. But I was able to pair them manually, so yes, thank you, solution already existed :)

2. Now that I think about it, you're right, this doesn't quite make sense.

3. Fair enough! The solution to 1 makes this less important imo anyway.
